From: Ben Hutchings <ben@decadent.org.uk>
Subject: aufs: Mark ad debian version
Date: Sun, 06 Oct 2013 23:34:37 +0100
Forwarded: not-needed

This was requested by the upstream author to make it clear that we've
cherry-picked fixes rather than importing one of his versions directly.

--- a/include/linux/aufs_type.h
+++ b/include/linux/aufs_type.h
@@ -39,7 +39,7 @@
 
 #include <linux/limits.h>
 
-#define AUFS_VERSION	"3.2.x"
+#define AUFS_VERSION	"3.2.x-debian"
 
 /* todo? move this to linux-2.6.19/include/magic.h */
 #define AUFS_SUPER_MAGIC	('a' << 24 | 'u' << 16 | 'f' << 8 | 's')
